garbage-collection-configuration.md

```diff
@@ -5,147 +5,35 @@ aliases: ['/docs-cn/dev/garbage-collection-configuration/','/docs-cn/dev/referen
-
-- `"legacy"`（默认）：使用旧的扫描方式，即关闭 Green GC。
-- `"physical"`：使用物理扫描的方式，即开启 Green GC。
-
-> **注意：**
-> 
-> 该项配置是隐藏配置。首次开启需要执行：
-> 
-> {{< copyable "sql" >}}
-> 
-> ```sql
-> insert into mysql.tidb values ('tikv_gc_scan_lock_mode', 'legacy', '');
-> ```
-
-## 关于 GC 流程的说明
-
-从 TiDB 3.0 版本起，由于对分布式 GC 模式和并行 Resolve Locks 的支持，部分配置选项的作用发生了变化。可根据下表理解不同版本中这些配置的区别：
-
-| 版本/配置          |  Resolve Locks          |  Do GC  |
-|-------------------|---------------|----------------|
-| 2.x               | 串行 | 并行 |
-| 3.0 <br/> `tikv_gc_mode = centered` <br/> `tikv_gc_auto_concurrency = false` | 并行 | 并行 |
-| 3.0 <br/> `tikv_gc_mode = centered` <br/> `tikv_gc_auto_concurrency = true` | 自动并行 | 自动并行 |
-| 3.0 <br/> `tikv_gc_mode = distributed` <br/> `tikv_gc_auto_concurrency = false` | 并行 | 分布式 |
-| 3.0 <br/> `tikv_gc_mode = distributed` <br/> `tikv_gc_auto_concurrency = true` <br/> （默认配置） | 自动并行 | 分布式 |
-
-表格内容说明：
-
-- 串行：由 TiDB 逐个向 Region 发送请求。
-- 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
-- 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
-- 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
-
-另外，如果 Green GC （实验特性）开启（即 [`tikv_gc_scan_lock_mode`](#tikv_gc_scan_lock_mode) 配置项设为 `"physical"`），Resolve Lock 的执行将不受上述并行配置的影响。
+* [`tidb_gc_enable`](/system-variables.md#tidb_gc_enable)
+* [`tidb_gc_run_interval`](/system-variables.md#tidb_gc_run_interval)
+* [`tidb_gc_life_time`](/system-variables.md#tidb_gc_life_time)
+* [`tidb_gc_concurrency`](/system-variables.md#tidb_gc_concurrency)
+* [`tidb_gc_scan_lock_mode`](/system-variables.md#tidb_gc_scan_lock_mode)
 
 ## 流控
 
-TiDB 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响，`0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
+TiDB 支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响，`0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
 
 {{< copyable "shell-regular" >}}
 
 ```bash
 tikv-ctl --host=ip:port modify-tikv-config -m server -n gc.max_write_bytes_per_sec -v 10MB
 ```
 
+## TiDB 5.0 引入的变化
+
+在 TiDB 5.0 之前的版本中，GC 是通过系统表 `mysql.tidb` 进行配置的。从 TiDB 5.0 版本起，GC 仍然可以通过系统表 `mysql.tidb` 进行配置，但建议你使用系统变量进行配置，这样可以确保对配置的任何更改都能得到验证，防止造成异常行为([#20655](https://github.com/pingcap/tidb/issues/20655))。
+
+TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC 的请求，因此不再提供 `CENTRAL` GC 模式的支持，取而代之的是效率更高的 `DISTRIBUTED` GC 模式 （自 TiDB 3.0 起的默认 GC 模式）。
+
+如果要了解 TiDB 历史版本中 GC 配置的变化信息，请使用左侧导航栏中的 _TIDB版本选择器_ 切换到本文档的历史版本。
+
 ## GC in Compaction Filter 机制
 
-GC in Compaction Filter 机制是在分布式 GC 模式 (distributed GC) 的基础上，由 RocksDB 的 Compaction 过程来进行 GC，而不再使用一个单独的 GC worker 线程。这样做的好处是避免了 GC 引起的额外磁盘读取，以及避免清理掉的旧版本残留大量删除标记影响顺序扫描性能。可以由 TiKV 配置文件中的以下开关控制：
+GC in Compaction Filter 机制是在分布式 GC 模式 (`DISTRIBUTED` GC) 的基础上，由 RocksDB 的 Compaction 过程来进行 GC，而不再使用一个单独的 GC worker 线程。这样做的好处是避免了 GC 引起的额外磁盘读取，以及避免清理掉的旧版本残留大量删除标记影响顺序扫描性能。可以由 TiKV 配置文件中的以下开关控制：
```

Comment:
I think you can write "`DISTRIBUTED` GC mode" (which uses code to describe the GC mode) or "Distributed GC" (which is a term), but "`DISTRIBUTED` GC" reads weird.

Reply:
Ok, "DISTRIBUTED GC mode" will be used.